pkg/agent/qrm-plugins/cpu/dynamicpolicy/mbm/strategy/relu_allocator.go

```diff
@@ -0,0 +1,115 @@
+/*
+Copyright 2022 The Katalyst Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package strategy
+
+// reluAllocator is the reLU style allocator to decide the shares among the participating numa nodes
+type reluAllocator struct{}
+
+func (r reluAllocator) AllocateDeductions(activeGroups GroupMBs, packageMB, mbThreshold float64) []map[int]float64 {
+	excessiveByPackage := packageMB - mbThreshold
```

Comment:
should we identify the case that excessiveByPackage is negtive and return a error ?

Reply:
not really needed here as this method is called only by mbm controller and the controller (caller) should make sure the condition is met. The same applies to AllocateIncreases method too. I'll add comments to interface signature to make this assumption sort of clear.